src/content/get-started/install/openshift.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Red Hat OpenShift
+description: How to install Okteto onto Red Hat OpenShift
+sidebar_label: Red Hat OpenShift
+id: openshift
+pagination_prev: get-started/install/index
+pagination_next: get-started/install-okteto-cli
+---
+
+import variables from '../../variables.json';
+import CodeBlock from '@theme/CodeBlock';
+
+This guide will walk you through the process of installing Okteto in Red Hat OpenShift.
+
+## Installation Requirements
+
+Before you start, make sure you have the following CLIs installed in your machine:
+
+- `kubectl` >= 1.28 ([kubectl installation guides](https://kubernetes.io/docs/tasks/tools/#kubectl))
+- `helm` >= 3.14 ([helm installation guides](https://helm.sh/docs/intro/install/))
+- `oc` >= 4.0.0 
+
+You'll also need the following:
+
+- An Okteto License
+- A Kubernetes cluster with OpenShift (≥ 4.0.0) installed
+
+For this guide, you will need to obtain the following values:
+
+- `<your-openshift-subdomain>`: for example, `okteto.r9yz.p2.openshiftapps.com`
+- `<cluster-name>`: for example, `okteto`
+
+## Getting your Okteto License
+
+A license is mandatory to use Okteto. You'll receive a license key as part of your subscription to Okteto. If you haven't received it, [please open a support ticket](https://okteto.com/support).
+
+If you are interested in evaluating Okteto, [sign up for our free 30 days trial](https://www.okteto.com/free-trial/). No credit card required.
+
+## Steps before Installing Okteto
```

Comment:
This "reads" odd.  Maybe we can reword it as "Before your start"  or "Prerequisites"? The closer we are to the language that the openshift guides use the better.